edge/pkg/edged/edged.go

```diff
@@ -320,6 +321,13 @@ func (e *edged) Start() {
 	// handled by pod workers).
 	go utilwait.Until(e.podKiller.PerformPodKillingWork, 5*time.Second, utilwait.NeverStop)
 
+	// update node label
+	node, _ := e.GetNode()
+	node.Labels = e.labels
+	if err := e.metaClient.Nodes(e.namespace).Update(node); err != nil {
```

Comment:
I'm afraid whether this updating will overide other lables setted from cloud like kubectl, would you like to have a test?

Reply:
thers is a problem, if we define customsiz labels like ```key1=value1``` in edgecore's configfile and we use kubectl label commonand to update label like ```key1=value2```,if we restart edgecore, the value of key1 label will become value1 again

Reply:
Yes, and here we should use patch to update the node spec(maybe can do it in cloudcore).

Reply:
sounds good,I have used patch to update node object